rubocop-airbnb/rubocop-airbnb.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
     'Gemfile',
   ]
 
-  spec.add_dependency('rubocop', '~> 0.58.0')
-  spec.add_dependency('rubocop-rspec', '~> 1.30.0')
-  spec.add_development_dependency('rspec', '~> 3.5')
+  spec.add_dependency('rubocop', '~> 0.62.0')
```

Comment:
The problem with doing this is that rules update as well. If you are 100% fine with the default provided by rubocop great, but otherwise this will start causing warnings of rules that are misnamed or misconfigured.

Reply:
Does this mean we'd have to have all the core rule names guarded behind a version check?

Reply:
do you know if these changes will work with the latest rubocop (0.69 - https://github.com/rubocop-hq/rubocop/commit/c2047b71a68d079533ebe9b3766a5ed80b7f8a42) ?